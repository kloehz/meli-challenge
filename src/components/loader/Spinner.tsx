import SpinnerSvg from "../../assets/images/Spinner.svg";

export const Spinner = () => {
    return (
        <div className="spinner-container">
            <img src={SpinnerSvg} className="spinner-image" />
        </div>
    )
}
