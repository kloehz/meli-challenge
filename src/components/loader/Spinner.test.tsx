import { shallow } from "enzyme"
import { Spinner } from "./Spinner"

describe('Spinner tests', () => {
    
    const wrapper = shallow(<Spinner />);

    test('should be render correctly', () => {
        expect(wrapper).toMatchSnapshot();
    });
    
    test('should be have an image', () => {
        expect(wrapper.find('img').length).toBe(1);
    });
    
})
