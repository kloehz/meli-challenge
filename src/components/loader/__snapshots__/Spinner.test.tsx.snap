// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner tests should be render correctly 1`] = `ShallowWrapper {}`;
